ee/maintained-apps/inputs/homebrew/intune-company-portal.json

```
@@ -0,0 +1,8 @@
+{
+  "name": "Intune Company Portal",
```

@marko-lisica 

osquery reports this as `Company Portal`, which seems like an obscure title.

We technically can use this name as the display name in the catalog, but after it's added and installed on a host, the title will change to the name on disk, which is `Company Portal`.  I imagine this might create a confusing customer experience if they are searching for "Intune" in software and nothing comes up.

One option could be to covert the osquery reported name to FMA name in inventory, but that may also cause confusion if an IT admin is looking for "Intune Company Portal" in the Finder on the host, but on disk is shows "Company Portal"

Another option could be to use the `Company Portal` name, but show the description text (and make the description searchable) in the catalog.

```suggestion
  "name": "Company Portal",
```